MEMBER_TRAVEL_FUND.md

```diff
@@ -0,0 +1,178 @@
+
+## Requirements
+
+* Candidates must be an active participant of one of the projects of the OpenJS Foundation.
+* Those requesting funds must indicate that they do not have funding available from another
+  source, such as an employer or the event itself that might cover costs for presenters.
+
+## Process
+
+Requests must be made prior to the start of travel. Requests submitted once travel is
+complete will not be approved.
+
+It is strongly recommended that you submit your request far enough in advance to get
+approval before making relevant expenditures. Review of requests will take at least 72
+hours and may take considerably longer. Note, with limited funds available, approval
+is not guaranteed. If a request is rejected, reimbursement will not be issued. The
+Foundation cannot pre-pay for travel expenditures.
+
+## Visa Tips
+
+For all events, 3rd party or hosted Foundation events, it is recommended to apply for
+a visa at least 6 months prior to an event.
+
+### For Industry Events
+
+If you need a visa for the country in which the OpenJS Foundation event you are
+attending is being held, [see the document on preparing for a visa application](https://github.com/openjs-foundation/cross-project-council/blob/master/TRAVEL_FUND/travel-visas.md).
+
+### What The Foundation Can Do To Help:
+
+The Foundation [provides guidance on preparing your visa application](https://github.com/openjs-foundation/cross-project-council/blob/master/TRAVEL_FUND/travel-visas.md),
+including information on visa processing times and a checklist of items that a
+visa application could require.
+
+For Foundation-hosted events, the OpenJS Foundation provides a visa letter by The
+Linux Foundation (does not guarantee visa approval). You can find the Visa Request
+page under `Attend > Visa Request`.
+
+### What the Foundation Cannot Do:
+
+Please note that unless the Foundation is specifically hosting the event, we cannot
+help with visa letters. The Foundation is also not able to contact consulates. When
+the Foundation has done this in the past, our efforts have been unsuccessful and/or
+we’ve lacked the expertise to be effective.
+
+For additional information on preparing your visa application or requesting a visa
+letter, please email [visainfo@openjsf.org](mailto:visainfo@openjsf.org).
+
+## Requesting Travel Funds
+
+* Open a pull request that adds an entry to the appropriate year's file
+  * Include as many fields as possible:
+    * The name of the event you plan to attend.
+    * The location, dates of the event and where you will be traveling from.
+    * The presentation you intend to give, if applicable.
+    * The size of the stipend you wish to receive.
+      * In the pull request description, explain how you calculated your
+        stipend. For example, include estimates for airfare, hotel, other
+        transportation, and so on. The total amount should be your stipend
+        request.
+      * In the PR description, mention @openjs-foundation/cpc.
```

Comment:
Just as a note, I don't think this actually pings CPC members unless the requester is a member of the org?